src/modules/navigation/tree/LoadingTree.tsx

```diff
@@ -2,11 +2,13 @@ import { Skeleton, Stack, Typography } from '@mui/material';
 
 const LoadingTree = (): JSX.Element => (
   <Stack p={1}>
-    {Array.from(Array(5)).map(() => (
-      <Typography>
-        <Skeleton variant="text" />
-      </Typography>
-    ))}
+    {Array.from(['elem 1', 'elem 2', 'elem 3', 'elem 4', 'elem 5']).map(
```

Comment:
I think you can remove the `Array.from`.

```suggestion
    {['elem 1', 'elem 2', 'elem 3', 'elem 4', 'elem 5'].map(
```